mkgu_packaging/dicarlo/marques/marques_ringach2002.py

```diff
@@ -0,0 +1,88 @@
+
+import numpy as np
+from brainio_collection.packaging import package_data_assembly
+import scipy.io as sio
+from brainio_base.assemblies import DataAssembly
+
+DATA_DIR = '/braintree/data2/active/users/tmarques/bs_datasets/Ringach2002.mat'
+ASSEMBLY_NAME = 'dicarlo.Marques2020_Ringach2002'
```

Comment:
as above, I believe the assembly identifier should be `hawken.Ringach2002`

Reply:
In this case, I agree with your suggestion because I extracted the data from Ringach's website.

Reply:
Changed to shapley.Ringach2002